### DISCUSSION TITLE
Disable AV1

### DISCUSSION THREAD

**Discussion by sryze** [mozilla-mobile/firefox-ios#18969] [Q&A]
Is it possible to disable AV1 support in Firefox for iOS? My device has been getting kind of hot lately when playing YouTube videos, I want to see if this is related. 

Thanks.

**Comment by Becau-hub** [comment]
> Is it possible to disable AV1 support in Firefox for iOS? My device has been getting kind of hot lately when playing YouTube videos, I want to see if this is related. 
> 
> 
> 
> Thanks.
